sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -102,18 +115,31 @@ WITH first_seen AS (
     normalized_channel AS channel,
     device_manufacturer,
     device_model,
-    normalized_os_version AS os_version
+    normalized_os_version AS os_version,
+    app_display_version AS app_version
   FROM
     `moz-fx-data-shared-prod.fenix.baseline_clients_first_seen`
   WHERE
     submission_date >= '2019-01-01'
     AND normalized_channel = 'release'
 ),
+-- Find activations per client_id. Data available since '2021-12-01'
+activations AS (
+  SELECT
+    client_id,
+    submission_date,
+    activated > 0 AS activated
+  FROM
+    `moz-fx-data-shared-prod.fenix.new_profile_activation`
+  WHERE
+    submission_date >= '2021-12-01'
+),
 -- Find earliest data per client from the first_session ping.
 first_session_ping AS (
   SELECT
     client_info.client_id AS client_id,
     MIN(sample_id) AS sample_id,
+    DATETIME(MIN(submission_timestamp)) AS min_submission_datetime,
```

Comment:
Is there a specific reason we don't just keep this as a timestamp?

Reply:
Yes, for analysis purposes we use the same format of submission_date, and already convert it here so that this conversion is not required in the view to retrieve the earliest `first_seen_date`.